codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/CustomizableOperationGenerator.kt

```diff
@@ -34,7 +35,7 @@ class CustomizableOperationGenerator(
     private val smithyTypes = CargoDependency.SmithyTypes(runtimeConfig).asType()
 
     fun render(crate: RustCrate) {
-        crate.withModule(RustModule.Operation) { writer ->
+        crate.withModule(RustModule.operation(Visibility.PUBLIC)) { writer ->
```

Comment:
I would have expected this to be enough to get the module to show up in `lib.rs` as `pub mod operation`, but apparently it wasn't - I had to add https://github.com/awslabs/smithy-rs/pull/1803/files#diff-f7da198414ffe3cefc591164d3b1bb41d67e22812d7ba3a86b8abd6d798cec49R87
I'd be curious to understand why.

Reply:
From reading the code, it definitely seems like it should emit the `mod` statement when it's missing from `DefaultPublicModules`.